index.js

```diff
@@ -47,7 +45,7 @@ endpoints.forEach((endpoint, i) => {
   pools.push(pool);
 
   routes.forEach((route) => {
-    const routePool = route.db === 'cache' ? leaderboardCache.cachePool : pool;
+    const routePool = route.db === 'cache' ? databaseCachePool : pool;
```

Comment:
Just a note: eventually we'll be removing testnet support from the server, so it'll just be a mainnet server + cache instead of this weird half-combo we have right now.